packages/webapp/public/locales/ml/translation.json

```diff
@@ -533,209 +533,209 @@
+        "INTERNAL_ERROR": "സെൻസർ \"{{ sensorId }}\" അപ്‌ലോഡ് പരാജയപ്പെട്ടിരിക്കുന്നു, പ്രശ്നപരിഹാരത്തിനായി ലൈറ്റ് ഫാം സപ്പോർട്ടിൽ support@litefarm.org എന്ന ഇമെയിലിൽ ബന്ധപെടുക."
       },
       "SENSOR_FIELDS": {
-        "BRAND": "",
-        "DEPTH": "",
-        "LATITUDE": "",
-        "LONGITUDE": "",
-        "MODEL": "",
-        "NAME": "",
-        "READING_TYPES": "",
-        "SENSOR_EXTERNAL_ID": ""
+        "BRAND": "ബ്രാൻഡ്",
+        "DEPTH": "ആഴം_സെ.മീ",
+        "LATITUDE": "അക്ഷാംശം",
+        "LONGITUDE": "രേഖാംശം",
+        "MODEL": "മോഡൽ",
+        "NAME": "പേര്",
+        "READING_TYPES": "വായന_തരങ്ങൾ",
+        "SENSOR_EXTERNAL_ID": "ബാഹ്യ_ഐഡി"
       },
-      "TITLE": "",
-      "UPLOAD_ERROR_LINK": "",
-      "UPLOAD_ERROR_MESSAGE": "",
-      "UPLOAD_INSTRUCTION_MESSAGE": "",
-      "UPLOAD_LINK_MESSAGE": "",
-      "UPLOAD_PLACEHOLDER": "",
+      "TITLE": "മാപ്പിലേക്ക് സെൻസറുകൾ ചേർക്കുക",
+      "UPLOAD_ERROR_LINK": "അവ കാണുന്നതിന് ഇവിടെ ക്ലിക്ക് ചെയ്യുക.",
+      "UPLOAD_ERROR_MESSAGE": "നിങ്ങളുടെ അപ്‌ലോഡിൽ ചില പ്രശ്‌നങ്ങളുണ്ടായി. ",
+      "UPLOAD_INSTRUCTION_MESSAGE": "ശരിയായ മാറ്റം വരുത്തുന്നതിനായി.",
+      "UPLOAD_LINK_MESSAGE": "രൂപരേഖ ഡൗൺലോഡ് ചെയ്യുക",
+      "UPLOAD_PLACEHOLDER": "csv ഫയൽ അപ്‌ലോഡ് ചെയ്യുക",
       "VALIDATION": {
-        "EXTERNAL_ID": "",
-        "FILE_ROW_LIMIT_EXCEEDED": "",
-        "MISSING_COLUMNS": "",
-        "SENSOR_BRAND": "",
-        "SENSOR_DEPTH": "",
-        "SENSOR_HARDWARE_VERSION": "",
-        "SENSOR_LATITUDE": "",
-        "SENSOR_LONGITUDE": "",
-        "SENSOR_MODEL": "",
-        "SENSOR_NAME": "",
-        "SENSOR_READING_TYPES": ""
+        "EXTERNAL_ID": "തെറ്റായ എക്സ്റ്റേർണൽ ഐഡി, ശരിയായ ഐഡിയിൽ 1നും 40നും ഇടയിൽ ക്യാറക്ടറുകൾ ഉണ്ടായിരിക്കണം.",
+        "FILE_ROW_LIMIT_EXCEEDED": "ഫയൽ വരിയുടെ പരിധി കവിഞ്ഞു. ഒരു ഫയലിൽ നിന്ന് അപ്‌ലോഡ് ചെയ്യാൻ കഴിയുന്ന സെൻസറുകളുടെ പരമാവധി എണ്ണം 100 ആണ്.",
+        "MISSING_COLUMNS": ". നിരകൾ ആവശ്യമാണ്/നഷ്‌ടമായി.",
+        "SENSOR_BRAND": "ഉത്പന്നതിന്റെ പേര് തെറ്റായി രേഖപെടുത്തിയിരിക്കുന്നു, 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ ഉണ്ടായിരിക്കണം.",
+        "SENSOR_DEPTH": "തെറ്റായ ആഴ കണക്ക്, ശരിയായ കണക്കിന് 0നും 1000നും ഇടയിൽ വരേണ്ടതാണ്.",
+        "SENSOR_HARDWARE_VERSION": "തെറ്റായ ഹാർഡ്‌വെയർ വിവരണ നാമം. ശരിയായതിൽ 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ ഉണ്ടായിരിക്കണം.",
+        "SENSOR_LATITUDE": "തെറ്റായ അക്ഷാംശരേഖ. -85നും 85. നും ഇടയിലും ദശാംശമുണ്ടെങ്കിൽ 10 ദശാംശത്തിൽ താഴേയും വേണ്ടതാണ്.",
+        "SENSOR_LONGITUDE": "തെറ്റായ ദേശാന്തരരേഖ. -180നും 180. നും ഇടയിലും ദശാംശമുണ്ടെങ്കിൽ 10 ദശാംശത്തിൽ താഴേയും വേണ്ടതാണ്.",
+        "SENSOR_MODEL": "തെറ്റായ മോഡൽ നാമം. ശരിയായതിൽ 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ വരേണ്ടതാണ്.",
+        "SENSOR_NAME": "തെറ്റായ സെൻസർ നാമം. ശരിയായതിൽ 1നും 100നും ഇടയിൽ ക്യാറക്ടറുകൾ വരേണ്ടതാണ്.",
+        "SENSOR_READING_TYPES": "തെറ്റായ വായന ഇനം (reading type) കണ്ടെത്തിയിരിക്കുന്നു: {{ reading_types }}. ശരിയായ മൂല്യങ്ങൾ ഇവയിൽ ഉൾപെടുത്തിയിരിക്കുന്നു: {{ allowed_reading_types }}."
```

Comment:
Is not translating the bracket here a mistake?

Reply:
No idea. Maybe it's a concept that makes more sense in English? I think it's okay to leave as it doesn't mess with the interpolated values.